src/Arcus.Testing.Tests.Integration/Storage/TemporaryBlobFileTests.cs

```diff
@@ -65,8 +64,10 @@ private async Task<TemporaryBlobFile> WhenBlobUploadedAsync(
 
             TemporaryBlobFile temp =
                 Bogus.Random.Bool()
+#pragma warning disable CS0618 // Type or member is obsolete: currently still testing deprecated functionality.
```

Comment:
We should still test deprecated code, so ignore these specific warnings on calls to our own marked deprecated code.